src/guides/share/groups/migrate-from-s3.rst

```diff
@@ -168,19 +169,17 @@ Timeline
 :March 2022: New groups created after this point manage their data through
              nextstrain.org instead of S3.
 
-:TBD: `Notification`_ sent to groups created prior to March.
+:early November: `Notification`_ sent to groups created prior to March.
 
-:TBD: Groups migrated one-by-one in coordination with group owners.
+:November onwards: Groups start migrating one-by-one in coordination with group owners.
 
-:TBD: Nextstrain team's desired deadline for migrating all groups.
+:end of February 2023: Nextstrain team's desired deadline for migrating all groups.
 
 
 Notification
 ============
 
-.. note:: Notifications have not yet been sent out.
-
-In *[TBD]* 2022, we emailed the contacts for all groups created before March 2022
+In November 2022, we emailed the contacts for all groups created before March 2022
 to inform them of these changes.  A copy of the email is below.
 
     **Subject:** Action requested: Improvements to how you manage your
```

Comment:
To me, the email doesn't convey the "Action requested" clearly – one has to click on the link and get through the What/Why sections to finally see the numbered list of requests.

I'd suggest adding a short sentence or two describing the migration, followed by the numbered list of requests without the details. This can then be followed by a link to the docs page to read more about it.

Reply:
Ah, so I intentionally omitted the `nextstrain remote` changes and an actual list of steps from the email. The idea was that I don't want folks to only read a summarized version in the email and not click thru because that will a) leave them without enough info and b) it isn't possible to update the email on-the-fly if we want to make clarifications as some migrations get started but others remain pending. (The "action requested" in the subject line was to indicate "This email requires a response", not so much as to describe the contents.)

Thoughts on balancing being too vague with trying to make sure folks click thru?

Prompted by your comment, I thought it good to shift things around ([3c14509](https://github.com/nextstrain/docs.nextstrain.org/pull/137/commits/3c14509e20548c3f00b91aae3345e4942d826a0b):

![image](https://user-images.githubusercontent.com/79913/199081985-42259691-5b36-47c9-aec8-bb895fc3a2c9.png)

But this still doesn't address your concern of not including an actual action item list.

Reply:
That looks good! My only concern for vagueness is lack of response. For those who don't respond the first time, we can send a reminder email that is more verbose.

Reply:
Ok, sounds good. Yeah, I expect we'll have to send some reminders regardless of what we do, but we'll see! :-)